gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_vertx.adoc

```diff
@@ -0,0 +1,24 @@
+[[vertx-scenario]]
+== Vert.x
+
+You can easily get started with using {plugin} on an https://vertx.io/[Eclipse Vert.x] without providing any explicit configuration. {plugin} would generate an opinionated container image and manifests by inspecting your project configuration.
+
```

Comment:
Maybe add the equivalent of 
https://github.com/eclipse/jkube/blob/c6539d789444f27a74e43e283ffac1891387e754/gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_minikube.adoc?plain=1#L43

> $ ./gradlew quarkusBuild k8sBuild k8sResource k8sApply